.github/workflows/main.yml

```diff
@@ -19,16 +19,29 @@ jobs:
     # conrainer with pre-installed mender-artifact and mender-cli tools
     container:
       image: mendersoftware/mender-ci-tools:mender-master
-    name: Upload a dummy Mender artifact
+    name: Deploy a dummy Mender Artifact
     steps:
       # Checkout git repository
       - uses: actions/checkout@v3
       # Download Mender bash client
+      - name: Serup environment
+        run: |
+          apk add openssl jq
```

Comment:
```suggestion
          apk add --no-cache openssl jq
```
?

Reply:
IMO it doesn't matter here because we don't build an image and care about its size, but here we just want to use the tools during the execution and all used storage is released afterwards.

Reply:
It is not (only) about the size.

The image might have an outdated cache and then `apk add` will fail unless you run `apk update` previously. So, adding the `--no-cache` option invalidates the local cache and ensures `apk add` will succeed.

See [this](https://stackoverflow.com/a/49119046) for example.

Reply:
Ah, right. That is good point! 👍